app/presenters/activity_presenter.rb

```diff
@@ -1,6 +1,14 @@
 class ActivityPresenter < BasePresenter
   presents :activity
 
+  def activity_day
+    h.local_date(activity.created_at.to_date, format: '%A, %B %e %Y', data: { behavior: 'activity-day-value' })
```

Comment:
Can we use `ACTIVITIES_STRFTIME_FORMAT` here?

Reply:
Yeah we can!